docs/use-cases/nft-auction/index.md

```diff
@@ -1,28 +1,37 @@
 
-The bidders run the same risk of bidding in a rigged auction but in the worst case, they can refuse to pay for the NFT. Anyhow, this setup would offer already a much better security story than all the fully custodial NFT drops done on Cardano today
+:::tip [Diagram for L1 transaction delays]
+:::
 
-:::info NOTES 
+However, the auction bidding mechanism is a perfect candidate for scaling on L2 via the Hydra technology, as transactions within a Hydra Head have short confirmation delays and immediate finality. A compelling L2-powered auction service can be built on Cardano if it is developed in a way that makes use of Hydra’s strengths.
 
-1. In this scenario, the Head is initialised by the auctioneer. It could be a "Managed Head" or "Head-as-a-Service", e.g. The auctioneer does not run a Hydra node but uses some third-party provider to run the node. A single-party head might not seem to make much sense but in this case it's just a way to do Cardano transactions with smart-contracts faster than on the mainchain.
+We envision that the Hydra-based auction framework will become a standard modular component that NFT marketplaces, games, and other Web 3.0 applications will plug into their architectures to add digital asset auctions to their product. Furthermore, it will stimulate a new business ecosystem for professional scalability providers to offer L2 hosting services to these applications, similar to the stakepool ecosystem and the emerging governance ecosystem.
 
-2. This use case is _extracted_ from a [conversation that happened on Github](https://github.com/input-output-hk/hydra/discussions/116). Have a look at the conversation for details and/or to comment.
-:::
+One possible path to realizing this vision could progress along the following set of milestones that we believe are feasible to implement using Hydra and its anticipated improvements:
+
+- Delegated voucher auction (invitational) – the first complete prototype of an auction that can host its bidding process on L2.
+
+- Delegated voucher auction (open + fully collateralized) – this would extend the initial prototype by enabling open auctions, where bidders can freely participate without permission from sellers. It would require bids to be fully collateralized upon submission.
+
+- Production-ready SDK for delegated voucher auctions – Prototypes could be made ready  for deployment as real-world services, wrapped in a software development kit (SDK).
+
+- Always-on delegated auction service (single-head) – This would allow a single persistent Hydra head to host multiple auctions over time without closing. Hydra Head operators would be able to build a business model offering L2-hosting services for auctions.
+
+- Always-on delegated auction service (multi-head) – This would allow an auction to split its bidding process among multiple Hydra Head hosts, reducing the reliance on any one Hydra Head.
```

Comment:
I think it should say "among multiple Hydra heads". You would have some degree of centralization in a delegated model. So choosing between different "service providers" (i.e. different delegated heads) would be a step more decentralized than a single service provider (even if it is multiple node operators running a single head - since they will likely be not too many, they can collude to censor bids).

I'll change it to "among multiple Hydra heads".